rotkehlchen/chain/ethereum/modules/aave/graph.py

```diff
@@ -209,6 +210,13 @@ class AaveEventProcessingResult(NamedTuple):
     total_earned_liquidations: Dict[Asset, Balance]
 
 
+def _get_version_from_reserveid(pairs: List[str], index: int) -> int:
+    """Gets the aave version from the reserve address id"""
+    if pairs[index] == '24a42fd28c976a61df5d00d0599c34c4f90748c8':
```

Comment:
Question: Where this constant comes from?

Reply:
check comment in next line. It's the address of v1 lending pool